lib/libs/preferences.dart

```diff
@@ -240,8 +251,118 @@ class Preferences extends SerializableObject with ChangeNotifier {
     _loggingEnabled = loggingEnabled;
   }
 
+  /// Prefs aus externer Datei laden
+  Future<String> loadFromExportJson(BuildContext context) async {
```

Comment:
Bitte die ganze Logik mit der Dateiauswahl und so in die `settings.dart` oder eine neue Datei schieben, ich würde gerne vermeiden, dass eine Funktion in einer der Datenquellen selbst eine Benutzerinteraktion fordert.

Reply:
Das JSON parsen kannst du schon hier machen, aber halt keine Dialoge bzw. Dateipicker anzeigen.